test/e2e/common.go

```diff
@@ -20,15 +20,28 @@ import (
 	"context"
 	"fmt"
 	"path/filepath"
+	"strings"
 
 	. "github.com/onsi/ginkgo"
 
+	"github.com/blang/semver"
+	"github.com/onsi/gomega/types"
 	corev1 "k8s.io/api/core/v1"
 	clusterv1 "sigs.k8s.io/cluster-api/api/v1alpha3"
 	"sigs.k8s.io/cluster-api/test/framework"
 	"sigs.k8s.io/cluster-api/util"
 )
 
+// Test suite constants for e2e config variables
+const (
+	KubernetesVersionCurrent              = "KUBERNETES_VERSION_CURRENT"
+	CNIPath                               = "CNI"
+	KubernetesVersionPrevious             = "KUBERNETES_VERSION_UPGRADE_FROM"
```

Comment:
```suggestion
	KubernetesVersionUpgradeFrom             = "KUBERNETES_VERSION_UPGRADE_FROM"
```
Let's keep const names consistent with variable names